Disable hostapd CONFIG_WPS_UPNP to mitigate CallStranger vulnerability

Since we are not using the WPS-UPNP functionality, the most
straightforward way to mitigate this vulnerability is to disable it.

All mitigation options:
- Disable WPS UPnP support in the hostapd build-time configuration
- Disable WPS UPnP support in the hostapd runtime configuration
- Apply hostapd patches from https://w1.fi/security/2020-1/
- Update to hostapd v2.10 or newer

Full details about hostapd CallStranger vulnerability and patches:
https://w1.fi/security/2020-1/upnp-subscribe-misbehavior-wps-ap.txt

--- a/bcmdrivers/broadcom/net/wl/impl69/main/components/opensource/router_tools/hostapd/hostapd/brcm.config
+++ b/bcmdrivers/broadcom/net/wl/impl69/main/components/opensource/router_tools/hostapd/hostapd/brcm.config
@@ -145,8 +145,8 @@ CONFIG_DPP=y
 
 # Wi-Fi Protected Setup (WPS)
 CONFIG_WPS=y
-# Enable UPnP support for external WPS Registrars
-CONFIG_WPS_UPNP=y
+# Disable WPS UPNP to mitigate CallStranger vulnerability
+CONFIG_WPS_UPNP=n
 # Enable WPS support with NFC config method
 CONFIG_WPS_NFC=y
 
--- a/bcmdrivers/broadcom/net/wl/impl69/main/components/opensource/router_tools/wpa_supplicant/wpa_supplicant/brcm.config
+++ b/bcmdrivers/broadcom/net/wl/impl69/main/components/opensource/router_tools/wpa_supplicant/wpa_supplicant/brcm.config
@@ -180,6 +180,8 @@ CONFIG_EAP_LEAP=y
 
 # Wi-Fi Protected Setup (WPS)
 CONFIG_WPS=y
+# Disable WPS UPNP to mitigate CallStranger vulnerability
+CONFIG_WPS_UPNP=n
 # Enable WPS external registrar functionality
 #CONFIG_WPS_ER=y
 # Disable credentials for an open network by default when acting as a WPS
